### ISSUE TITLE
how to deactivate demultiplexing by qiime?

### ISSUE THREAD

**Issue by golrokh51** [nf-core/ampliseq#103]
I have already demultiplexed data as input and I get the following error:
` [7e/b5b5a2] process > qiime_demux_visualize         [100%] 1 of 1, failed: 1 ✘`
`Execution cancelled -- Finishing pending tasks before exit`
`ERROR ~ Error executing process > 'qiime_demux_visualize (demux)'`

**Comment by d4straub** [comment]
Hey there, did you intentionally close it?
Anyway, this pipeline isn't demultiplexing, the `demux` is a admittedly weird naming from QIIME that I just kept, this process produces quality plots.

**Comment by golrokh51** [comment]
Hey! Yes I did. I am running the pipeline on a cluster with Slurm. I am using singularity profile offline as my nodes have no access to internet. After solving about all my problems, the program gives new errors each time. The first time it was `demux` and I thought this was the problem. But just by rerunning the job without changing anything, I saw that this error went away. So I understood that the problem is not the QIIME. Thanks.
